std_msgs/Duration wait_duration
---
bool answer